package.json

```diff
@@ -50,9 +50,24 @@
       "protobufjs@>=6.10.0 <7.2.4": ">=7.2.4",
       "@babel/traverse": ">=7.23.2",
       "follow-redirects": ">=1.15.4",
-      "web3": ">=4.2.1",
-      "web3-core": ">=4.2.1",
-      "web3-utils": ">=4.2.1"
+      "@openzeppelin/contracts@>=4.0.0 <4.9.3": ">=4.9.3",
+      "postcss@<8.4.31": ">=8.4.31",
+      "zod@<=3.22.2": ">=3.22.3",
+      "@openzeppelin/contracts@>=4.7.0 <4.9.2": ">=4.9.2",
```

Comment:
i'm guessing this is the result of the pnpm audit fix some time ago. if you rebase, we can probably get away with not having to make these changes for now :)